screensaver.video/resources/collections/aquarium.xml

```diff
@@ -0,0 +1,41 @@
+<videoscreensaver>
+	<collection>Aquarium</collection>
+	<encoded>true</encoded>
+	<video>
+		<name>Aquarium 1 - [846MB] - 400p</name>
+		<filename>Aquarium001.mkv</filename>
+		<image>http://kodi.wiki/images/4/45/Aquarium001.png</image>
+		<duration>3602</duration>
+		<primary>aHR0cDovL2tvZGkuemlnZ3k3MzcwMS5zZWVkci5pby9WaWRlb1NjcmVlbnNhdmVyL0FxdWFyaXVtMDAxLm1rdg==</primary>
+		<secondary>aHR0cDovLzE3OC4zMy42MS42L3B1dHN0b3JhZ2UvRG93bmxvYWRGaWxlLmFzaHg/RG93bmxvYWRGaWxlSGFzaD03QTcyNTkyQTNBNUE0QTVRUVdFMjAxMTY2N0VXUVM=</secondary>
+	</video>
+	<video>
+		<name>Aquarium 2 - [2.7GB] - 720p</name>
+		<filename>Aquarium002-720p.mkv</filename>
+		<image>http://kodi.wiki/images/4/44/Aquarium002-720p.png</image>
+		<duration>3602</duration>
+		<primary>aHR0cDovL2tvZGkuemlnZ3k3MzcwMS5zZWVkci5pby9WaWRlb1NjcmVlbnNhdmVyL0FxdWFyaXVtMDAyLTcyMHAubWt2</primary>
+	</video>
+	<video>
+		<name>Aquarium 3 - [2.4GB] - 720p</name>
+		<filename>Aquarium003-720p.mp4</filename>
+		<image>http://kodi.wiki/images/d/df/VideoScreensaver-Aquarium003.PNG</image>
+		<duration>7200</duration>
+		<primary>aHR0cDovL2tvZGkuemlnZ3k3MzcwMS5zZWVkci5pby9WaWRlb1NjcmVlbnNhdmVyL0FxdWFyaXVtMDAzLTcyMHAubXA0</primary>
+	</video>
+	<video>
+		<name>Aquarium 4 - [780MB] - 720p</name>
+		<filename>Aquarium004-720p.mp4</filename>
+		<image>http://kodi.wiki/images/3/39/VideoScreensaver-Aquarium004.JPG</image>
+		<duration>2048</duration>
+		<primary>aHR0cDovL2tvZGkuemlnZ3k3MzcwMS5zZWVkci5pby9WaWRlb1NjcmVlbnNhdmVyL0FxdWFyaXVtMDA0LTcyMHAubXA0</primary>

```

Comment:
please understand one of the goals of team kodi is to promote **open source** software.
code obfuscation doesn't fit that vision i'm afraid.

the addon review process is to make sure we don't distribute illegal content (or links to it) through our addon repository. by obfuscating the url, it's impossible for us to determine what videos you link to.
at least without going through all kind of hoops to decode it.